File: CorePlay/Models/DataModel/ComitteeContact.m
//
//  ComitteeContact.m
//  AlKadi
//
//  Created by Ghafar Tanveer on 14/02/2017.
//  Copyright © 2017 Khurram Bilal Nawaz. All rights reserved.
//

#import "ComitteeContact.h"

@implementation ComitteeContact

+(BOOL)propertyIsOptional:(NSString *)propertyName {
    
    return YES;
}
+(JSONKeyMapper*)keyMapper
{
    NSDictionary * dict = @{ @"Data"                        : @"contacts"
                             };
    
    return [[JSONKeyMapper alloc]initWithModelToJSONDictionary:dict];
    //return [[JSONKeyMapper alloc] initWithDictionary:dict];
}
@end

File: CorePlay/Models/DataModel/ComitteeContact.swift
//
//  ComitteeContact.swift
//  CourPlay
//
//  Created by Ghafar Tanveer on 24/04/2017.
//  Copyright © 2017 Ghafar Tanveer. All rights reserved.
//

import Foundation
import JSONModel

class ComitteeContact: JSONModel {
   
    var contacts:NSArray
     = []
    override class func keyMapper() -> JSONKeyMapper {
        
        let dict = ["contacts"   : "Data"]
        return JSONKeyMapper(modelToJSONDictionary: dict)
    }
    override class func propertyIsOptional(_ propertyName:String!) -> Bool{
        if(propertyName == "dasda"){
            return true
        }
        return false
    }
}
